dx/utils/tracking.py

```diff
@@ -120,15 +96,28 @@ def get_df_variable_name(
     """
     Returns the variable name of the DataFrame object.
     """
+    logger.debug("looking for matching variables for dataframe")
```

Comment:
Curious. What is the use case for this utility?

Reply:
This was previously for the blueprintjs callout to the user after a resample request that returned instructions to pass the filtered subset to a new variable, like:
```
new_df = my_awesome_df.query( < pandas query string >, engine="python")
```
But it's going to be passed in the metadata to the frontend to let DEX handle displaying the query syntax dynamically when the filters change, instead of just on (backend) resample requests. I'd like to have this handled in the dataframe caching branch for easier access.